docs/source/Security_Services_Capability_Maturity_Index/System_Integrity_Requirements.md

```diff
@@ -216,6 +200,5 @@ Implement default-deny technologies (such as AppLocker) to only permit applicati
 
 Applies to: Vendor provided hardware
 
-Method: Derived
 
 Reference: CIS EIAT 17
```

Comment:
@mike-garcia Did you want the EIAT removed here? IIRC, you wanted all trace of it gone, yes?

Reply:
YRC correctly, let's get rid of it. Future of EIAT is uncertain and I'd rather not risk the confusion